src/tools/perf/lib/ucp_tests.cc

```diff
@@ -432,13 +469,17 @@ class ucp_perf_test_runner {
                 ucx_perf_update(&m_perf, 1, length);
                 ++sn;
             }
+
+            wait_last_iter(recv_buffer, 1);
```

Comment:
no need to pass "length" parameter at all to wait_last_iter, send_last_iter - they can just use '1'

Reply:
done